src/promptflow/promptflow/_sdk/_constants.py

```diff
@@ -236,7 +236,7 @@ def get_list_view_type(archived_only: bool, include_archived: bool) -> ListViewT
 VIS_LIB_CDN_LINK_TMPL = (
     "https://sdk-bulk-test-endpoint.azureedge.net/bulk-test-details/view/{version}/bulkTestDetails.min.js?version=1"
 )
-VIS_LIB_VERSION = "0.0.29"
+VIS_LIB_VERSION = "0.0.31"
```

Comment:
might need to confirm with UX dev, will 0.0.31 be back compatible with old contract?

tagging @RealKai42

Reply:
yes
but it should be 0.0.30, @PeiwenGaoMS

Reply:
I know, I haven't changed here yet.

Reply:
Confirmed with Kaiyi, it is `0.0.31`.